kan/docker-compose.yml

```
@@ -0,0 +1,26 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: kan_server_1
+      APP_PORT: 3000
+  server:
```

I would disable the proxy auth here as kan comes with authentication out of the box.
```suggestion
      APP_PORT: 3000
      PROXY_AUTH_ADD: "false"

  server:
```